README.md

```diff
@@ -1,97 +0,0 @@
-# Product Name: Money Wise
```

Comment:
Why are we deleting the readme?

Reply:
Not sure. Likely got lost in the reorg. Adding it back.